controller/novels.js

```diff
@@ -0,0 +1,11 @@
+const models = require('../models')
+
+const getAllNovels = async (request, response) => {
+  const novels = await models.novels.findAll()
```

Comment:
Missing this line to make to accurate call for 2nd to last get in the README 
 include: [{ model: models.Authors }, { model: models.Genres }]